### DISCUSSION TITLE
Did not RECEIVE my XCH

### DISCUSSION THREAD

**Discussion by Valentin373737** [Chia-Network/chia-blockchain#17602] [Support]
Hello. I sent XCH from the OKX exchange to my CHIA wallet, but the coins never reached/

XCH
5,20875363 XCH
0,0004 XCH
Chia
OKX
Address: xch1275pzf45dhczc8t2cy3xy52925r8a3lyncaar3jccezcfdjd9m0sywd5nj
TXID: 0xe6eccd71973b3c83116cbeb378191051869b76d332cf649ba27c0babfdecc653

**Comment by Valentin373737** [comment]
the issue has been resolved, the money has been received

**Reply by BrandtH22** [reply]
Thank you for the confirmation and make it an excellent day! 🌱
